src/lib/utils/sqlite_wrapper.hpp

```diff
@@ -31,33 +30,50 @@ class SQLiteWrapper final {
    * @param table      The table to load into sqlite
    * @param tablename  The desired table name
    */
-  void create_table(const Table& table, const std::string& table_name);
+  void create_sqlite_table(const Table& table, const std::string& table_name);
 
-  /*
-   * Executes a sql query in the sqlite database context and returns a Hyrise table.
-   *
-   * @param sql_query Query to be executed
-   * @returns An opossum Table containing the results of the executed query
-   */
-  std::shared_ptr<Table> execute_query(const std::string& sql);
+  class Connection final {
+   public:
+    explicit Connection(const std::string& uri);
+
+    Connection(const Connection&) = delete;
+    Connection(Connection&&) noexcept;
+    Connection& operator=(const Connection&) = delete;
+    Connection& operator=(Connection&&) = delete;
+    ~Connection();
+
+    sqlite3* db{nullptr};
+
+    /*
+     * Executes a sql query in the sqlite database context and returns a Hyrise table.
+     *
+     * @param sql_query Query to be executed
+     * @returns An opossum Table containing the results of the executed query
+     */
+    std::shared_ptr<Table> execute_query(const std::string& sql) const;
+
+    /**
+     * Execute an SQL statement on the wrapped sqlite db without invoking any Hyrise parts
```

Comment:
Emphasize: "Executes a **single** SQL statement..."

Reply:
Where did you pick up that this is the case? You could also send multiple statements.

Reply:
You're right! 

> The sqlite3_exec() interface runs zero or more UTF-8 encoded, semicolon-separate SQL statements passed into its 2nd argument, in the context of the database connection passed in as its 1st argument.

I concluded this (by mistake) from `SQLiteWrapper::Connection::execute_query(const std::string& sql)` where we pass each SQL statement to SQLite individually.